docs/test-pages/TCD-003.adoc

```diff
@@ -0,0 +1,23 @@
+= TCD-003
+
+== Purpose
+
+Verifying the statement definition rules in the IDE (Eclipse or idea).
+
+=== Tested requirements
+
+REQ-SYNT-002
+
+=== Used JSL model
+
+xref:resources/shop2.jsl[shop2.jsl]
+
+== Test scenario
+
+. Open the xref:resources/shop2.jsl[shop2.jsl] with the IDE.
+
+. The IDE is handling and showing the opened file as a JSL file (Syntax highlight is being OK.), and there isn't any error.
```

Comment:
"is OK"

Reply:
"there aren't any"

Reply:
@noherczeg 
My English knowladge the correct format are the follows: 
"there isn't any error"
OR
"there are no errors"

Reply:
I fixed to this: "there are no errors"